sdk/identity/azure-identity/azure/identity/aio/_credentials/shared_cache.py

```diff
@@ -65,20 +66,28 @@ async def get_token(self, *scopes: str, **kwargs: Any) -> AccessToken:  # pylint
         if not scopes:
             raise ValueError("'get_token' requires at least one scope")
 
-        if not self._initialized:
-            self._initialize()
+        if not self._client_initialized:
+            self._initialize_client()
+
+        is_cae = bool(kwargs.get("enable_cae", False))
```

Comment:
why not  is_cae = kwargs.get("enable_cae", False)?

Reply:
Since I pass this in as a param of type `bool` in later functions, the type checker doesn't like that it could potentially be of type `Any`